README.md

```diff
@@ -104,6 +104,11 @@ Once you have set up the projects you want to work on, and want to run all proje
 * go to [https://local.ft.com:5050/syndication/user-status](https://local.ft.com:5050/syndication/user-status) to confirm the app is responding with data
 * Optionally, you can also run `npm run monit` to bring up the [PM2 process monitor](https://www.npmjs.com/package/pm2#cpu--memory-monitoring) e.g. for checking out CPU and memory usage.
 
+### Running against prod DB (optional)
+To run `next-syndication-api` agains prod DB you just have to add to the environment variables defined in `custom-environment-variables.yaml` the prefix `_PROD`.  
```

Comment:
[Typo] `agains` is missing the T at the end.
[Nitpick] As you're changing this, could you please change this to read 
> To run `next-syndication-api` against prod DB the only change you need to do is to add the prefix `_PROD` to the environment variables defined in `custom-environment-variables.yaml`.

Nota bene: please try to avoid "just" (as well as "simple" in the docs) - the gist is that when one says “just do x” or “this is simple” it isn’t great for newcomers for whom “just do x” might mean weeks of learning how to do X, ditto for “simple”.